packages/field-plugin/src/messaging/pluginMessage/pluginToContainerMessage/GetContextMessage.ts

```diff
@@ -1,12 +1,20 @@
 import { isMessageToContainer, MessageToContainer } from './MessageToContainer'
 
-export type GetContextMessage = MessageToContainer<'getContext'>
+export type GetContextMessage = MessageToContainer<'getContext'> & {
+  // Previously, debounced message was the default behavior.
+  // That debouncing implementation can be problematic, for example,
+  // when multiple field plugin instances request for context.
+  debounce: false
```

Comment:
Never saw this before, is the debouncing implemented inside the storyfront?

Reply:
@BibiSebi yes, but only for `requestContext`. The debouncing is enabled by default for `requestContext`, but neither Johannes nor I know why it exists. And it seems troublesome especially with multiple field plugin instances. So I've added this new property to bypass debouncing, which is used only for the Field Plugin SDK (not the legacy ones).